### DISCUSSION TITLE
Switch to less obnoxious stakepool explorer

### DISCUSSION THREAD

**Discussion by zopsicle** [Emurgo/yoroi-mobile#1880] [Ideas]
In the iOS app, when going to Staking Center and tapping the title of any pool, it opens adapools.org, which is full of animated ads, pop-ups, and flashy/pulsating buttons. It’s obnoxious and may scare users away because it looks scammy.

Do you know of any other Cardano stakepool explorers that could be used instead? Something calm and uncluttered like the Cardano transaction explorer: https://explorer.cardano.org/en

![screenshot](https://user-images.githubusercontent.com/50083900/125762207-40a68d0d-c929-44b5-ae73-43189780b3d1.png)

**Comment by mikeymop** [comment]
Maybe [pooltool](https://pooltool.io), or [pool.pm](https://pool.pm/iog/stake) could be parsed and used in the delegation screen?
